src/SortableContainer/index.js

```diff
@@ -55,8 +55,8 @@ export default function sortableContainer(WrappedComponent, config = {withRef: f
       lockToContainerEdges: false,
       lockOffset: '50%',
       getHelperDimensions: ({node}) => ({
-        width: node.offsetWidth,
-        height: node.offsetHeight,
+        width: node.getBoundingClientRect().width,
+        height: node.getBoundingClientRect().height,
       }),
```

Comment:
Similar to @hledley's suggesion, this should only call `getBoundingClientRect` one
```suggestion
      getHelperDimensions: ({node}) => node.getBoundingClientRect()),
```